sources/dialect/README.MD

```diff
@@ -0,0 +1,19 @@
+# Dialect Support
+
+This rule has net yet been added.
+For now, we have only added some files and sources.
+This section will gradually be expanded as the rule is written.
+
+The files inside ``original_files`` were added from the version 1 tree of the original repository, [``en-wl/wordlist``](https://github.com/en-wl/wordlist/tree/v1).
```

Comment:
You should explain why you are using the v1 and not the v2. I assume it's for the license that is more open.

I mean people might wonder why you are using a years old version, and/or would like to us to upgrade

Reply:
The newer version is only 2 weeks old it seems that doesn't yet include varcon. I've added a brief explanation.